Modified our-sponsors page with 2025 sponsors

jaipurandare opened the pull request:
Updated the 2025 Sponsors according to the [google doc](https://docs.google.com/document/d/1DZjNdHgiI5OMys6hjlus1c-tiO3LfqyDGyWTPkL7rb4/edit?tab=t.0)

Not sure of the url for Meta. So used [meta-loves-python](https://engineering.fb.com/2024/02/12/developer-tools/meta-loves-python/)

There are no sponsors in Gold.  Used the "Your logo here" image as place holder. 

Renamed the Community section to Team so that it is consistent with the sponsorship table. Removed the Diversity section.

Changed the sponsorship table on "Sponsor Us" page to image tag instead of iframe because it is not going to change. Constant image will be available to all public across browsers. Issue created for this [https://github.com/pybay/PyBay/issues/54](https://github.com/pybay/PyBay/issues/54)

jaipurandare committed:
Modified our-sponsors page with 2025 sponsors

jaipurandare committed:
Changed the community sponsor to Team as per the sponsorship table

jaipurandare committed:
Changed the sponsorship table from iframe to image tag

surfaceowl reviewed (changes requested):
@jaipurandare 

A few updates please:
1. meta url.  the current URL points to an old blog post - I will follow up with them, but for now, please use this url instead:  https://engineering.fb.com/
2. Team level does not replace Community level -- Community and Diversity are two separate levels that are not found in the table because they have few sponsorship benefits and we wanted to keep the table clean.  However, Community and Diversity are found in our Sponsorship  Prospectus on the last page....   Please Restore Community level.... So we must definitely have the following levels:  Gold, Silver, Bronze, Team, Community and Diversity....

jaipurandare committed:
Added back community and diversoty sponsor blocks and related css

jaipurandare commented:
Changed the Meta url to https://engineering.fb.com/

Added back the Community and Diversity sponsor sections
Attaching screenshots. 
Is it ok to have the "Your logo here" image for the sections Gold, Team and Diversity? We don't have sponsors for those sections right now. 
<img width="1440" alt="Screenshot 2025-05-29 at 20 06 13" src="https://github.com/user-attachments/assets/7f0da6dc-46d5-435f-9853-ae93c6ee2e11" />
<img width="1440" alt="Screenshot 2025-05-29 at 20 06 25" src="https://github.com/user-attachments/assets/5f923af1-313f-4b27-a45f-45f72acd4324" />
<img width="1440" alt="Screenshot 2025-05-29 at 20 06 29" src="https://github.com/user-attachments/assets/c006ba18-8b81-4a04-a71b-70086672507c" />
<img width="1440" alt="Screenshot 2025-05-29 at 20 06 33" src="https://github.com/user-attachments/assets/ed046603-beea-43ef-816b-4c3aaf541aee" />
<img width="1440" alt="Screenshot 2025-05-29 at 20 06 35" src="https://github.com/user-attachments/assets/425716ff-e709-49ee-9496-f48cbc841ef0" />
<img width="1440" alt="Screenshot 2025-05-29 at 20 06 40" src="https://github.com/user-attachments/assets/c6dcd409-b61e-4555-8352-ef0a67a082af" />

surfaceowl reviewed (approved):
@jaipurandare  -- this looks great - and thanks for the quick turnaround.  Nice job!
